images/core-image-minimal-rubygems.bb

```diff
@@ -0,0 +1,21 @@
+require recipes-core/images/core-image-minimal.bb
```

Comment:
potential issue: Please add an 'SPDX-License-Identifier: MIT' comment to the file